### PR TITLE
Refreshing SSL certificates

### LINKED ISSUE #1060: Expired SSL certificates and how to renew them
Ceph services use SSL certificates for secure communication. Their validity is temporary and has to be renewed in case it expires.

### PR DESCRIPTION
**Pull request by tbazant**
Fixes #1060

### CODE REVIEW

File: xml/admin_saltcluster.xml [hunk 1]
```diff
@@ -1771,4 +1770,209 @@ election_strategy: 1
...
   </sect1>
+  <sect1 xml:id="refreshing-ssl-certificates">
+    <title>Refreshing expired SSL certificates</title>
+
+    <para>
+      Multiple &ceph; services use SSL certificates to secure communication
+      between the client and the server. Validity of an SSL certificate is
+      normally limited and expires after a time period specified at its
+      creation. Following are procedures to renew SSL certificates for affected
+      &ceph; services.
```
Comment (dariavladykina) [inline xml/admin_saltcluster.xml:1781]:
```suggestion
      between the client and the server. The validity of an SSL certificate is
      normally limited and expires after the time period specified at its
      creation. The following are procedures to renew SSL certificates for
      affected &ceph; services.
```

File: xml/admin_saltcluster.xml [hunk 2]
```diff
@@ -1771,4 +1770,209 @@ election_strategy: 1
...
+    </para>
+
+    <tip>
+      <para>
+        The following procedures start with <emphasis>renewing</emphasis> an
+        expired certificate. By <emphasis>renewing</emphasis>, we mean
+        obtaining a valid certificate and key file with the expiration time in
+        future. A certificate authority (CA) can provide one for you, or you
+        can self-sign the certificate yourself.
```
Comment (dariavladykina) [inline xml/admin_saltcluster.xml:1790]:
```suggestion
        the future. A certificate authority (CA) can provide one for you, or you
        can create a self-signed certificate yourself.
```

File: xml/admin_saltcluster.xml [hunk 3]
```diff
@@ -1771,4 +1770,209 @@ election_strategy: 1
...
+          </para>
+<screen>&prompt.cephuser;ceph orch restart <replaceable>NAME_OF_RGW_SERVICE</replaceable></screen>
+        </step>
+      </procedure>
+      <note>
+        <para>
+          If you originally deployed the SSL certificate by specifying the
+          <option>rgw_frontend_ssl_certificate</option> option in the &ogw;
+          specification file, delete it from the specification to avoid having
+          two different certificates specifications.
```
Comment (dariavladykina) [inline xml/admin_saltcluster.xml:1858]:
```suggestion
          two different certificate specifications.
```

File: xml/admin_saltcluster.xml [hunk 4]
```diff
@@ -1771,4 +1770,209 @@ election_strategy: 1
...
+      <para>
+        Renewing the &grafana; SSL certificate is almost identical to initial
+        SSL certificate setup mentioned in
+        <xref linkend="monitoring-alerting"/>.
+      </para>
+      <itemizedlist>
+        <listitem>
+          <para>
+            If you are using a self-signed certificate, remove the existing one
+            from the ceph configuration and reconfigure the &grafana; service
```
Comment (dariavladykina) [inline xml/admin_saltcluster.xml:1910]:
```suggestion
            from the &ceph; configuration and reconfigure the &grafana; service
```

File: xml/admin_saltcluster.xml [hunk 5]
```diff
@@ -1771,4 +1770,209 @@ election_strategy: 1
...
+</screen>
+        </listitem>
+      </itemizedlist>
+    </sect2>
+
+    <sect2 xml:id="renewing-ssl-certificates-grafana">
+      <title>&grafana;</title>
+      <para>
+        Renewing the &grafana; SSL certificate is almost identical to initial
+        SSL certificate setup mentioned in
```
Comment (dariavladykina) [inline xml/admin_saltcluster.xml:1903]:
```suggestion
        Renewing the &grafana; SSL certificate is almost identical to the
        initial SSL certificate setup mentioned in
```

**Review by dariavladykina — COMMENTED** [review]
Hi, please see some suggestions. Thanks!

File: xml/admin_saltcluster.xml [hunk 6]
```diff
@@ -1771,4 +1770,210 @@ election_strategy: 1
...
+      affected &ceph; services.
+    </para>
+
+    <tip>
+      <para>
+        The following procedures start with <emphasis>renewing</emphasis> an
+        expired certificate. By <emphasis>renewing</emphasis>, we mean
+        obtaining a valid certificate and key file with the expiration time in
+        future. A certificate authority (CA) can provide one for you, or you
+        the future. A certificate authority (CA) can provide one for you, or
```
Comment (tserong) [inline xml/admin_saltcluster.xml:1790]: Odd partially duplicate line here.

**Review by tserong — APPROVED** [review]
LGTM, minus one odd partially duplicate line :-)

File: xml/admin_saltcluster.xml (hunk 6 above)
Comment (tbazant) [inline xml/admin_saltcluster.xml:1790]:
```suggestion
        the future. A certificate authority (CA) can provide one for you, or
```

**Pull request merged**